programs/zstdcli.c

```diff
@@ -205,6 +205,7 @@ static void usage_advanced(const char* programName)
     DISPLAYOUT( "--long[=#]: enable long distance matching with given window log (default: %u) \n", g_defaultMaxWindowLog);
     DISPLAYOUT( "--fast[=#]: switch to very fast compression levels (default: %u) \n", 1);
     DISPLAYOUT( "--adapt : dynamically adapt compression level to I/O conditions \n");
+    DISPLAYOUT( "--[no-]row-matchfinder : force enable/disable usage of fast row-based matchfinder for greedy, lazy, and lazy2 strategies \n");
```

Comment:
minor nit: Why not `--row-match-finder`? It seems inconsistent to use a dash between `row-match` but not `matchfinder`.

Reply:
I had been thinking of "matchfinder" as a single word (like in the enum ZSTD_useRowMatchfinderMode_e), but yeah.. it is two words.